src/drivers/flydigi_vader_4_pro/hid_report_test.rs

```diff
@@ -0,0 +1,35 @@
+use std::error::Error;
+
+use packed_struct::PackedStructSlice;
+
+use super::hid_report::PackedInputDataReport;
+
+#[tokio::test]
+async fn test_flydigi_vader_4_pro() -> Result<(), Box<dyn Error>> {
+    let report = PackedInputDataReport::unpack_from_slice(&DATA_A).unwrap();
+    println!("{report}");
+    assert!(report.a, "should be pressed");
+
+    Ok(())
+}
```

Comment:
Since this is not a valid passing test, you should remove this file or create a working test.